test/test_heuristic.cpp

```diff
@@ -142,10 +220,16 @@ TEST_P(HeuristicTest5Q, Identity) {
   ibmqYorktownMapper->map(settings);
   ibmqYorktownMapper->dumpResult(GetParam() + "_heuristic_qx4_identity.qasm");
   ibmqYorktownMapper->printResult(std::cout);
+  auto& result = ibmqYorktownMapper->getResults();
+  EXPECT_LE(result.heuristicBenchmark.effectiveBranchingFactor,
+            result.heuristicBenchmark.averageBranchingFactor);
```

Comment:
just for my understanding: this is just a sanity check, right?

Reply:
Yes exactly, if the effective branching rate is higher than the average branching rate, something went wrong in the benchmarking. On the one hand, adding those lines was just a large number of "free tests" for the benchmarking (the two branching factors also rely on all other benchmark values except for `timePerNode`), and on the other hand, if something went wrong there one should know, that they should not trust the benchmarking results of those tests.

Reply:
Seems fairly reasonable!

Reply:
Good idea. I did that now. Thank you!